Lib/fontTools/varLib/__init__.py

```diff
@@ -262,7 +262,7 @@ def _add_gvar(font, masterModel, master_ttfs, tolerance=0.5, optimize=True):
 		endPts = control.endPts
 
 		for i,(delta,support) in enumerate(zip(deltas[1:], supports[1:])):
-			if all(abs(v) <= tolerance for v in delta.array) and not isComposite:
+			if all(v == 0 for v in delta.array) and not isComposite:
```

Comment:
the `tolerance` parameter of `_add_gvar` seems to be no longer used, did you leave it in intentionally (e.g. not to break other code)? In that case, default it to `None` and do `warnings.warn` if set by caller.

Reply:
The `tolerance` argument seems to be still used in the IUP optimizer. So I left it as is.

Reply:
Ah, I missed that one (that chunk of code was hidden), thanks